apps/crn-server/src/controllers/research-outputs.ts

```diff
@@ -376,6 +377,7 @@ export interface ResearchOutputController {
   fetchById: (id: string) => Promise<ResearchOutputResponse>;
   create: (
     researchOutputRequest: ResearchOutputCreateData,
+    published?: boolean,
```

Comment:
I think it should be 
```suggestion
    publish?: boolean,
```
the past tense just doesn't make much sense to me, what do you think?

Reply:
In fact maybe better if it was an object with a property `publish` ? That way if we ever need to add more options to the this method we don't need to add another argument, just expand this one. I would call this object CreateOptions or something

Reply:
fixed here [review: change published to create options](https://github.com/yldio/asap-hub/pull/2667/commits/b072177f672bd5f5e7ce1a2eeea1d168638ed9ce)